geomesa-utils/pom.xml

```diff
@@ -84,6 +88,10 @@
             <artifactId>gt-data</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-shapefile</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
*blinks* I wonder if we could use a geomesa-shapefile module or something to line up dependencies, etc.